spec/index.html

```diff
@@ -648,7 +646,8 @@ <h2>Streams</h2>
     </section>
 
     <p>
-      Each of the individual <a>YAML documents</a> in the stream has to be converted into a <a>JSON-LD document</a> and
+      Each of the individual <a>YAML documents</a> in the stream
+      has to be converted into a separated <a>JSON-LD document</a> and
```

Comment:
Not that I necessarily agree with this requirement, but I think this sounds better.

```suggestion
      MUST be converted into a separate <a>JSON-LD document</a> and
```

Reply:
I think it's not a requirement, but the only thing you can do. While you can convert the YAML-LD stream to a JSON sequence, every document can only be converted to a separate JSON document.

We could define a way to "merge" all the YAML-LD documents in a stream into a single JSON-LD document, but this is quite complex to do reliably.

Reply:
I think we discussed that this is similar to how the [JSON-LD API HTML content algorithms](https://www.w3.org/TR/json-ld11-api/#html-content-algorithms) deal with multiple script elements. Specifically, step 6.2:

> Otherwise, the [extractAllScripts](https://www.w3.org/TR/json-ld11-api/#dom-loaddocumentoptions-extractallscripts) option is true. Set document to a new empty [array](https://infra.spec.whatwg.org/#list). For each [JSON-LD script element](https://www.w3.org/TR/json-ld11-api/#dfn-json-ld-script-element) in input:

> 1. Set source to its [textContent](https://dom.spec.whatwg.org/#dom-node-textcontent).
> 2. Set script content to the result of the [Extract Script Content algorithm](https://www.w3.org/TR/json-ld11-api/#extract-script-content), using source, rejecting promise with a [JsonLdError](https://www.w3.org/TR/json-ld11-api/#dom-jsonlderror) whose code set from the result, if an error is detected and processing is terminated.
> 3. If script content is an [array](https://infra.spec.whatwg.org/#list), merge it to the end of document.
> 4. Otherwise, append script content to document.

Reply:
I'll file another issue and leave the current text as is in this PR.